gasp-syncer/compose.yml

```diff
@@ -20,8 +20,8 @@ services:
       - SOURCE_WS_URL=${ETH_WS_URL:-ws://eth-stub:8545}
       - TARGET_RPC_URL=${ETH_RPC_URL:-http://arbitrum-stub:8546}
       - TARGET_WS_URL=${ETH_WS_URL:-ws://arbitrum-stub:8546}
```

Comment:
The TARGET rpc and ws urls should be ARB urls llike the arbitrum-stub - they shouldn't be ETH urls